test/runtests.jl

```diff
@@ -45,10 +45,10 @@ end
 
 A = [1 0; 0 -2]
 F = eigfact(Positive, A)
-if VERSION < v"v0.5-"
-    @test_approx_eq full(F) abs(A)
-else
+if isdefined(:≈)
     @test full(F) ≈ abs.(A)
```

Comment:
looks like 0.4 is having problems with the `abs.(A)`, which using `@compat` would help with since that is a syntax rewrite